reflex-dom-core/src/Foreign/JavaScript/TH.hs

```diff
@@ -289,12 +289,10 @@ class Monad m => MonadJS x m | m -> x where
   getJSProp :: String -> JSRef x -> m (JSRef x)
   withJSNode :: Node -> (JSRef x -> m r) -> m r
 
-type HasJS' = HasJS JS'
-
 #ifdef ghcjs_HOST_OS
 
 data JSCtx_IO
-type JSX = JSCtx_IO
+type JS' = JSCtx_IO
```

Comment:
https://github.com/reflex-frp/reflex-dom/pull/294/files#diff-2ad803dcbd54993a98c189f742f8f78eR299 I think I just created `JS'` for sake of `HasJS'`?